app/assets/src/components/views/samples/samples_view.scss

```diff
@@ -73,31 +73,7 @@
 
     .action {
       &:not(:first-child) {
-        margin-left: 20px;
-      }
-
-      .icon {
-        fill: $light-grey;
-        flex: 0 0 auto;
-        height: 21px;
-        width: 21px;
-
-        &:not(.disabled) {
-          cursor: pointer;
-          fill: $primary-light;
-        }
-
-        &.save {
-          width: 21px;
-        }
-
-        &.heatmap {
-          margin-left: 4px;
-        }
-
-        &.download {
-          width: 28px;
-        }
+        margin-left: 2px;
```

Comment:
Why the reduction?

Reply:
This is because the size of the whole icon is much bigger now. Up from 21px to 42px. To include the gray hover circle. So the margin needed to be reduced.